content/en/security/cloud_security_management/guide/active-protection.md

```diff
@@ -44,27 +44,27 @@ To check if Active Protection is already enabled in your organization, go to [Ag
 
 {{< img src="security/cws/guide/protection-column.png" alt="The protection column indicates that Active Protection is enabled in the org" style="width:100%;" >}}
 
-If Active Protection is available for a rule, then **Monitoring** or **Blocking** is listed in the **Protection** column.
+If Active Protection is available for a crypto mining rule, then **Monitoring** or **Blocking** is listed in the **Protection** column.
 
-If there is no **Monitoring** or **Blocking** in the **Protection** column, then Active Protection is not available for that rule yet.
+If there is no **Monitoring** or **Blocking** in the **Protection** column, then Active Protection is not available for that crypto mining rule yet.
 
-When Active Protection is enabled, and applies to a rule that generated a signal, you can see it by doing the following:
+When Active Protection is enabled, and applies to a crypto mining rule that generated a signal, you can see it by doing the following:
 
 1. In [Signals][1], open a signal.
 2. In the signal, view **Next Steps**. 
    - If Active Protection is enabled, in **Proactively block threats**, the **Active Protection Enabled** is displayed.
    - If Active Protection is not enabled, **Active Protection Enabled** is not displayed.
 
-If Active Protection is enabled and available for an Agent rule, you can see it by looking at the rule:
+If Active Protection is enabled and available for an Agent crypto mining rule, you can see it by looking at the rule:
 
-1. In [Agent Configuration][2], select a rule.
-2. In the rule, if Active Protection is enabled and available, there is a **Protection** section.
+1. In [Agent Configuration][2], select a crypto mining rule.
+2. In the crypto rule, if Active Protection is enabled and available, there is a **Protection** section.
```

Comment:
```suggestion
2. In the crypto mining rule, if Active Protection is enabled and available, there is a **Protection** section.
```
Just checking if it was intentional to shorten to "crypto rule" for step 2? Everywhere else you changed the text to "crypto mining rule".